### DISCUSSION TITLE
Can't verify collage mail id for github student pack.

### DISCUSSION THREAD

**Discussion by Manik2607** [community/community#112527] [GitHub Education]
### Select Topic Area

Question

### Body

i am trying to sign up for github student pack but i cant verify my collage mail id. i am recving verificaton mails but when i click verify button i am getting redirected to email settings with this error "There was an error verifying your email."

**Comment by adarsh-site** [comment]
If you're encountering issues with verifying your college email for the GitHub Student Pack, there are a few steps you can take to troubleshoot:

Check Email Settings: Ensure that your email settings, particularly any filters or rules you may have set up, are not interfering with the verification process. Make sure the verification email isn't being marked as spam or automatically sorted into another folder.

Try Different Browsers/Devices: Sometimes, issues can arise due to compatibility or caching problems with your current browser or device. Try accessing the verification link from a different browser or device to see if that resolves the issue.

Contact Support: If the problem persists, reach out to GitHub Support for assistance. They can provide guidance specific to your situation and help troubleshoot any technical issues you may be experiencing with email verification.

Check College Email Settings: Occasionally, email verification issues can stem from restrictions or filters set up by your college's email system. Check with your college's IT department to ensure that there are no restrictions preventing the verification process from completing successfully.

Wait and Retry: Sometimes, verification processes encounter temporary glitches. If you've tried the above steps and are still experiencing issues, wait for some time and then try again later. It's possible that the problem may resolve itself over time.

By following these steps, you should be able to troubleshoot and resolve the issue with verifying your college email for the GitHub Student Pack. If the problem persists, don't hesitate to reach out to GitHub Support for further assistance.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Manik2607! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
